server/cluster/unsafe_recovery_controller.go

```diff
@@ -34,6 +35,11 @@ import (
 
 type unsafeRecoveryStage int
 
+const (
+	storeReportRequestInterval = time.Second * 60
+	storePlanRequestInterval   = time.Second * 60
```

Comment:
Can we use merge them into one?

Reply:
+1

Reply:
I'll do it in the next PR, let's merge it first